packages/elvis-dropdown-lit/src/e-option.ts

```diff
@@ -0,0 +1,76 @@
+import { getThemeColor } from '@elvia/elvis-colors';
+import { getTypographyCss } from '@elvia/elvis-typography';
+import { consume } from '@lit/context';
+import { LitElement, css, html, unsafeCSS } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { classMap } from 'lit/directives/class-map.js';
+
+import { type DropdownContext, dropdownContext } from './dropdown-context';
+
+@customElement('e-option')
+export class EOption extends LitElement {
+  static styles = css`
+    :host {
+      height: var(--option-height);
+      display: flex;
+      align-items: center;
+
+      ${unsafeCSS(getTypographyCss('text-md'))};
```

Comment:
This makes sense. It would be very cool if we could build these styles into the component through a "build" so that we could have `@elvia/elvis-typography` as a dev dependency